app/serializers/student_comment_index_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class StudentCommentIndexSerializer < BaseSerializer
+
+    attributes :id, :content, :updated_at
```

Comment:
@maxwolffe I think you might have four spaces instead of two spaces in these files (not sure why that changed)??